lib/src/screens/reciters/reciters_screen.dart

```diff
@@ -242,9 +243,13 @@ class RecitersScreen extends ConsumerWidget {
                                 decoration: BoxDecoration(
                                   image: DecorationImage(
                                     fit: BoxFit.cover,
-                                    image: CachedNetworkImageProvider(
-                                      data[indexInPage].image!,
-                                    ),
+                                    image: UniversalPlatform.isWeb
+                                        ? Image.network(
+                                            data[indexInPage].image!,
```

Comment:
Why not using cachedNetworkImageProvider

Reply:
I think the problem is in the CachesImageNetwork but the problem is CORS